exporters/geneva/src/exporter.cc

```diff
@@ -78,50 +78,85 @@ opentelemetry::sdk::common::ExportResult Exporter::Export(
       for (auto &point_data_with_attributes : metric_data.point_data_attr_) {
         size_t body_length = 0;
         if (nostd::holds_alternative<sdk::metrics::SumPointData>(
-                point_data_with_attributes.point_data)) {
+                point_data_with_attributes.point_data)) 
+        {
           auto value = nostd::get<sdk::metrics::SumPointData>(
               point_data_with_attributes.point_data);
-          MetricsEventType event_type = MetricsEventType::ULongMetric;
-          if (nostd::holds_alternative<double>(value.value_)) {
+          ValueType new_value = value.value_;
+
+          MetricsEventType event_type = MetricsEventType::Undefined;
```

Comment:
This assignment is not needed as event_type is always assigned below？

Reply:
Yes, removed the assignment now.